lib/views/desktop/receive/receive.dart

```diff
@@ -11,7 +11,7 @@ import 'package:flutter_screenutil/flutter_screenutil.dart';
 import '../../shared/receive.dart';
 import '../../desktop/receive/widgets/DTReceivingDone.dart';
 import '../../desktop/receive/widgets/DTReceiveProgress.dart';
-import '../../mobile/receive/widgets/ReceiveConfirmation.dart';
+import '../../desktop/receive/widgets/DTReceiveConfirmation.dart';
```

Comment:
Should use `DTReceiveConfirmation` below

Reply:
Nevermind, I guess the naming is just not consistent. I saw the added class doesn't use `DT` as a prefix.